GPU/Common/FramebufferManagerCommon.h

```diff
@@ -356,8 +356,6 @@ class FramebufferManagerCommon {
 	bool MayIntersectFramebuffer(u32 start) const {
 		// Clear the cache/kernel bits.
 		start &= 0x3FFFFFFF;
-		if (Memory::IsVRAMAddress(start))
-			start &= 0x041FFFFF;
```

Comment:
How will this properly detect a block transfer between a VRAM mirror in RAM (i.e. 0x04600000 let's say) and RAM (i.e. 0x09000000)?  We now correctly normalize framebuffers and depth buffers to always mask out the mirrors so they're not all over the place.

-[Unknown]

Reply:
I am sorry but I have no knowledge of that code.

Removal of this two lines fixed the mentioned issue for me.